types/react-google-picker/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for react-google-picker 0.1
+// Project: https://github.com/sdoomz/react-google-picker
+// Definitions by: Evan Broder <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/react-google-picker/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me